src/azul/strings.py

```diff
@@ -70,3 +70,28 @@ def splitter(sep: Optional[str] = None, maxsplit: int = -1):
     ['a', 'b c']
     """
     return lambda s: s.split(sep, maxsplit)
+
+
+def trunc_ellipses(string: str, max_len: int) -> str:
+    from azul import (
```

Comment:
I guess `reject` is in the wrong module.

Reply:
Should I address this in this PR? Or open a ticket?

Reply:
https://github.com/DataBiosphere/azul/issues/2743

Reply:
We agreed to go with Hannes's idea during parking lot.